PMM-8355 Set TTL Monitor to 60s to make cluster come up

jprukner opened the pull request:
TTL monitor 60s is set in official example [cr.yaml](https://github.com/percona/percona-server-mongodb-operator/blob/v1.9.0/deploy/cr.yaml), it was cause of our cluster not coming up.

Commit:
PMM-8355 Set TTL Monitor to 60s to make cluster come up

Commit:
fix deps

oter commented on tests/go.mod:
```diff
@@ -9,11 +9,11 @@ go 1.16
 	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7 // indirect
 	golang.org/x/sys v0.0.0-20201029080932-201ba4db2418
-	google.golang.org/grpc v1.36.1
+	google.golang.org/grpc v1.38.0
```
is it needed for the task to update grpc version?

jprukner replied:
it is not, but result of `go mod tidy`, I would need to do it anyway. Is there some issue with the version?

jprukner replied:
Whatever, I reverted it, updated only dbaas-api which was blocking the tests.

oter replied:
Technically update of other deps should be in a separate PR. You could use `go get github.com/percona-platform/dbaas-api@branch-name` for updating just needed dependency.

oter replied:
fixed

jprukner replied:
It turned out it is needed. Because dbaas-api uses it. I suppose all `go mod tidy` additions are needed for this to build.

Commit:
Revert "fix deps"

This reverts commit ccf3500163bff65cdfa895799fa1683b1a52276f.

Commit:
update just dbaas api to make api tests pass

Commit:
fix deps again

Commit:
fix deps for real now

Commit:
Merge remote-tracking branch 'origin/main' into PMM-8355-support-psmdb-1-9-0

Commit:
hardcode psmdb operator 1.9.0

Commit:
Don't install operators in subtest

This can lead to timing error - subtests are run before PSMDB operator is installed

Commit:
Merge branch 'main' into PMM-8355-support-psmdb-1-9-0

Commit:
update kubernetes version

Commit:
fix liter complaining
